moveit_setup_assistant/src/widgets/start_screen_widget.cpp

```diff
@@ -305,6 +305,13 @@ void StartScreenWidget::loadFilesClick()
   }
   else
   {
+    if ( !create_new_package_ && config_data_->author_name_.empty() && config_data_->author_email_.empty() )
+    {
+      QMessageBox::warning( this, "Loading Project Without Author Information",
+        "The loaded configuration package does not include author information.\n"
+        "Make sure you regenerate the '.setup_assistant' and 'package.xml' files to add it.");
```

Comment:
On Tue, Aug 30, 2016 at 01:31:36AM -0700, Robert Haschke wrote:
> When loading an existing package, `package.xml` might already contain valid author information. With this hint you suggest to overwrite this!

Yes. Many packages do *not* contain valid information in the `package.xml` and they are the reason for this whole author-information addition.
Do you have a suggestion how to improve the message?